CHANGELOG/CHANGELOG-1.2.md

```diff
@@ -58,27 +58,27 @@
 
 filename | sha256 hash
 -------- | -----------
-[kubernetes.tar.gz](https://storage.googleapis.com/kubernetes-release/release/v1.2.7/kubernetes.tar.gz) | `53db157923c17fa7a0addb3e4dfe7d1b9194b9266a87d371a251d5bb790a1832`
-[kubernetes-src.tar.gz](https://storage.googleapis.com/kubernetes-release/release/v1.2.7/kubernetes-src.tar.gz) | `e6e46831706743d8263581d0575507cf5ffc265096d22e5e84cf1c3ae925db5e`
```

Comment:
We should change these super old changelogs, however this does prove that we've been advertising with dl.k8s.io since 1.10 or so.

I think the plan for now is to leave kubernetes-release available but convert it to read-only once we can migrate to using a bucket under a community account. So these can stay, but we want to remove _current_ references.

Reply:
I've reverted the CHANGELOG updates.